packages/wrangler/src/__tests__/parse.test.ts

```diff
@@ -0,0 +1,272 @@
+import {
+  formatMessage,
+  searchLocation,
+  indexLocation,
+  parseJSON,
+  parseTOML,
+} from "../parse";
+
+describe("formatMessage", () => {
+  test.each([
```

Comment:
Given that the actual test is so simple (just a single expectation check) there is no value in defining these tests like this, rather than just having a series of `it()` clauses. Moreover, if you do it with separate `it()` clauses then you can also make use of `matchesInlineSnapshot()` helpers.

Reply:
Also, as you mention in the comments, testing the formatting generated by the esbuild formatter is not very valuable to us. I think it would be enough just to check that these strings contain the information that we want using `contains()` or `matches()` helpers.